dockerfiles/igv/makeigvpesr_cram.py

```diff
@@ -46,9 +46,9 @@ def ped_info_readin(ped_file):
         pin = line.strip().split()
         if not pin[1] in out.keys():
             out[pin[1]] = [pin[1]]
-        if not(pin[2]) == 0:
+        if not (pin[2]) == 0:
```

Comment:
If you're going to adjust this for style, this line is very problematic. pin is an array of strings, so `not (pin[1]) == 0` always evaluates to True. If the goal is to only append non-trivial keys, it should be changed to:
```suggestion
        if pin[2] != "0":
```
Otherwise the if statement should be removed.
Similarly with pin[3]